src/field_ground_effect.c

```diff
@@ -100,17 +226,17 @@ void GetGroundEffectFlags_Tracks(struct MapObject *mapObj, u32 *flags)
     {
         *flags |= 0x100;
     }
-    else if (MetatileBehavior_IsSandOrDeepSand(mapObj->mapobj_unk_1F)
-     || MetatileBehavior_IsUnusedFootprintMetatile(mapObj->mapobj_unk_1F))
```

Comment:
We normally put the `&&` and `||` on the next line, so you didn't need to change this.

Reply:
This was how `clang-format` decided to format it. Should I not be running that before committing?